core/app/presenters/spree/variant_options_presenter.rb

```diff
@@ -0,0 +1,47 @@
+module Spree
```

Comment:
```suggestion
module Spree
  module Variants
     class OptionsPresenter
```

`core/app/presenters/spree/variants/options_presenter.rb`